### PR TITLE
Added a PHP to pull data from `objects` table, CSSed it

### PR DESCRIPTION
**Pull request by Liliya-V**

### CODE REVIEW

File: portfolio.php [hunk 1]
```diff
@@ -26,16 +28,19 @@
...
             </nav>
             <div class="works">
-                <h3>
-                    Let me please introduce you my works.
-                </h3>
-                    <img class="demo" src="images/lamp1.png" alt="lamps">
-                    <img class="demo" src="images/lamp2.png" alt="lamps">
-                    <img class="demo" src="images/lamp3.png" alt="lamps">
+                <?php
+                echo '<h3>' . 'My projects:' . '</h3>';
```
Comment (snack-attack) [inline portfolio.php]: don't need to use string concatenation here

File: portfolio.php [hunk 2]
```diff
@@ -26,16 +28,19 @@
...
+                                <a href="' . $project['link'] . '"><img class="demo" src="'. $project['image'] . '" alt="lamp"></a>
+                                <h5>'.$project['title'].'</h5>
+                             </div>';
+                }
+                ?>
             </div>
         </div>
         <div class="homecenter">
-            <img class="arrows aboutme-arrows" src="images/arrow-img.png" alt="Arrows">
+<!--            <img class="arrows aboutme-arrows" src="images/arrow-img.png" alt="Arrows">-->
```
Comment (snack-attack) [inline portfolio.php]: delete commented out code before merging

File: portfolio.php (hunk 1 above)
Comment (Liliya-V) [reply portfolio.php]: done

File: portfolio.php [hunk 3]
```diff
@@ -26,16 +24,18 @@
...
             </nav>
             <div class="works">
-                <h3>
-                    Let me please introduce you my works.
-                </h3>
-                    <img class="demo" src="images/lamp1.png" alt="lamps">
-                    <img class="demo" src="images/lamp2.png" alt="lamps">
-                    <img class="demo" src="images/lamp3.png" alt="lamps">
+                <?php
+                echo '<h3>My projects:</h3>';
```
Comment (mporam) [inline portfolio.php]: if your not using a php variable, you dont need to put your HTML inside PHP. just put it as normal HTML outside the PHP tags

File: portfolio.php [hunk 4]
```diff
@@ -26,16 +24,18 @@
...
+                foreach ($projects as $project) {
```
Comment (mporam) [inline portfolio.php]: what if `$projects` is empty?

**Review by mporam — CHANGES_REQUESTED** [review]

File: portfolio.php [hunk 5]
```diff
@@ -26,16 +24,23 @@
...
-                </h3>
-                    <img class="demo" src="images/lamp1.png" alt="lamps">
-                    <img class="demo" src="images/lamp2.png" alt="lamps">
-                    <img class="demo" src="images/lamp3.png" alt="lamps">
+                <h3>My projects:</h3>
+                <?php
+
+                   if (empty($projects)) {
+                       echo 'There are no projects yet';
+                       } else {
```
Comment (mporam) [inline portfolio.php]: fix indentation

**Review by mporam — CHANGES_REQUESTED** [review]

**Review by mporam — APPROVED** [review]

**Pull request merged**